tICA/README.md

```diff
@@ -0,0 +1,19 @@
+# Temporal ICA
+
+This pipeline uses ICA on a large amount of fMRI data (typically a large group of subjects),
+in order to separate global nuisance effects (such as due to respiration) from neural signals that
+have a nonzero spatial mean.
+
+The processing modes also allow for applying temporal ICA cleanup to datasets that are not
+large enough to reliably produce a new temporal ICA decomposition.  Briefly, the processing
+modes are:
+
+- NEW - do a full pipeline run from scratch, starting with a fresh MIGP, intended for large datasets
+
+- REUSE_SICA_ONLY - use a previous MIGP result, but estimate the temporal ICA from scratch
+
+- INITIALIZE_TICA - use a previous MIGP result, and estimate temporal ICA starting from a previous
+tICA mixing matrix, but allow it to change to fit the new data
+
+- REUSE_TICA - use a previous MIGP and tICA mixing matrix (and signal/nuisance classification),
+intended for small datasets
```

Comment:
Wouldn't this also represent the mode if you were just trying to process some additional subjects in a dataset that had already been processed?

Reply:
Ideally, if they use the same protocols, etc, you would probably want to just run all of them together when you are done with all of your acquisition.  If you treat a part of the group as a separate dataset, then "small datasets" covers that case.  I don't feel the need to enumerate everything that could be considered a "small dataset".

Reply:
So, what is the recommended approach to adding a "small" number of subjects (acquired with same protocol) to tICA processing that has already been completed.  Would be nice for the `README` to provide guidance on that situation.

Reply:
Probably depends on your processing resources, schedule, and how "small" the number of added subjects is.  The more data you use in the decomposition, the better the ICA is, subject to some kind of diminishing returns, probably.  @glasserm ?

Reply:
You could either initialize or use depending upon whether you wanted to recompute everything or just quickly clean those subjects.